themes/bootstrap5/templates/Recommend/SideFacets/multiFacetsSelection.phtml

```diff
@@ -1,5 +1,8 @@
+<?php
+$attributes = $this->multiFacetsSelectionValue === 'only' ? 'style="display: none;"' : '';
+?>
 <div class="multi-filters-selection">
-  <div>
+  <div <?= $attributes ?>>
```

Comment:
If you move the blank space from before the `<?=` to inside the attributes string above, then you can avoid having `<div >` in the generated code for some cases.